source/Government.h

```diff
@@ -49,6 +49,13 @@ class Government {
 	// Get the color to use for displaying this government on the map.
 	const Color &GetColor() const;
 	
+	// Get the government's tribute hails
+	const Phrase *TributeIgnoredHail() const;
```

Comment:
It may be worthwhile to take a page from the Government::GetHail() book, and use a single method with an int type specifier to determine which hail to return.

e.g. 
```
Government::GetTributeHail(int tributeStatus)

static const tribute[5] = { *tributeIgnoredHail, ..... *tributeSurrenderedHail }
return tribute[tributeStatus]->Get()
```
(or however)